### PR TITLE
Bugfix/hclecomm base page fix

### PR DESCRIPTION
**Pull request by DeepaliAgrwl**
Thumbnail has been added to the hclecomm pages (hecleomm base page. us page and en page). Please verify

### CODE REVIEW

File: ui.content/src/main/content/jcr_root/content/hclecomm/_jcr_content/image/file.dir/.content.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     jcr:primaryType="nt:file">
     <jcr:content
         jcr:lastModifiedBy="admin"
-        jcr:mimeType="image/png"
+        jcr:mimeType="image/jpeg"
```
Comment (rishabh-behl) [inline ui.content/src/main/content/jcr_root/content/hclecomm/_jcr_content/image/file.dir/.content.xml:6]: @DeepaliAgrwl : If you are uploading .png then why this has been changed to jpeg.

**Review by rishabh-behl — COMMENTED** [review]
@DeepaliAgrwl : Pl check single comment

File: ui.content/src/main/content/jcr_root/content/hclecomm/_jcr_content/image/file.dir/.content.xml (hunk 1 above)
Comment (DeepaliAgrwl) [reply ui.content/src/main/content/jcr_root/content/hclecomm/_jcr_content/image/file.dir/.content.xml:6]: @rishabh-behl the thumbnail image shared with Ashish was .jpg image. That is why the image mime type is changed.

Comment (DeepaliAgrwl) [reply ui.content/src/main/content/jcr_root/content/hclecomm/_jcr_content/image/file.dir/.content.xml:6]: The mime type of hcleocmm-content-page-template thumbanail is also image/jpeg itself

**Pull request merged**